erpnext/crm/doctype/opportunity/opportunity.js

```diff
@@ -170,12 +174,31 @@ erpnext.crm.Opportunity = frappe.ui.form.Controller.extend({
 		});
 	},
 
+	setup_fields: function() {
```

Comment:
I think better we move this into `set_dynamic_field_label` and change the function name to `update_dynamic_fields`